scipy/stats/_continuous_distns.py

```diff
@@ -9244,6 +9244,22 @@ def _entropy(self, kappa):
         return (-kappa * sc.i1e(kappa) / sc.i0e(kappa) +
                 np.log(2 * np.pi * sc.i0e(kappa)) + kappa)
 
+    @extend_notes_in_docstring(rv_continuous, notes="""\
+        The default limits of integration are endpoints of the interval
+        of width ``2*pi`` centered at `loc` (e.g. ``[-pi, pi]`` when
+        ``loc=0``).\n\n""")
+    def expect(self, func=None, args=(), loc=0, scale=1, lb=-np.pi, ub=np.pi,
```

Comment:
I think this is what was intended, given the notes above.
```suggestion
    def expect(self, func=None, args=(), loc=0, scale=1, lb=None, ub=None,
```